best_practices/forms.rst

```diff
@@ -42,14 +42,9 @@ form in its own PHP class::
                 'data_class' => 'AppBundle\Entity\Post'
             ));
         }
-
-        public function getName()
-        {
-            return 'post';
-        }
     }
 
-To use the class, use ``createForm`` and instantiate the new class::
+To use the class, pass the class name to the ``createForm()`` method::
```

Comment:
I personally dislike `FQN` acronym because it's not that well known in non-English speaking countries.